cypress/apiRequests/Order.js

```diff
@@ -78,3 +82,30 @@ export function getOrder(orderId) {
   }`;
   cy.sendRequestWithQuery(query).its("body.data.order");
 }
+
+export function fulfillOrder({ orderId, warehouse, quantity, linesId }) {
+  let lines = "";
+  linesId.forEach(lineId => {
+    lines += `{orderLineId:"${lineId.id}"
+      stocks:{
+        quantity:${quantity}
+        warehouse:"${warehouse}"
+      }
+    }`;
+  });
```

Comment:
There's a method on Array type called `reduce` that does exactly the thing you want - takes an array and outputs (usually) non-Array variable. Here's how you use it:
```
// reduce((accumulator, currentValue) => { ... }, initialValue)

linesId.reduce((lines, lineId) => {
  line = `...`;
  return lines + line;
}, "")
```

Reply:
I replaced forEach with reduce